### PR TITLE
gh-97669: Remove outdated example scripts

### PR DESCRIPTION
**Pull request by vstinner**
Remove outdated example scripts. Copy them from Python 3.11 if you still need them.

Removed scripts:

* byext.py
* byteyears.py
* cleanfuture.py
* copytime.py
* crlf.py
* db2pickle.py
* dutree.doc
* dutree.py
* find-uname.py
* find_recursionlimit.py
* finddiv.py
* findlinksto.py
* findnocoding.py
* fixcid.py
* fixdiv.py
* fixheader.py
* fixnotice.py
* fixps.py
* get-remote-certificate.py
* google.py
* highlight.py
* ifdef.py
* import_diagnostics.py
* lfcr.py
* linktree.py
* lll.py
* mailerdaemon.py
* make_ctype.py
* mkreal.py
* objgraph.py
* pdeps.py
* pickle2db.py
* pindent.py
* pysource.py
* reindent-rst.py
* rgrep.py
* suff.py
* texi2html.py
* which.py

Changes:

* Remove test_fixcid, test_lll, test_pdeps and test_pindent of test.test_tools.
* Remove get-remote-certificate.py changelog entry, since the script was removed.
* FYI there is a copy of crlf.py in Lib/test/test_lib2to3/data/.

* Issue: gh-97669

### CODE REVIEW

**Comment by vstinner** [comment]
Here is a first PR to remove scripts which look outdated to me. The remaining ones might be useful or are referenced in other parts of Python, so I prefer to write separated PRs for the other scripts that I plan to remove.

Since my first version, I restored md5sum.py and parse_html5_entities.py removed by mistake. I alsot restored untabify.py since it's used by patchcheck.py.

Note: first I created PR #97674 on the branch name "main" which is not convenient, so I closed it and created this PR instead.

**Comment by ericsnowcurrently** [comment]
It would be good to preserve at least some of these before this PR (or other PRs) get merged.  See my post at https://discuss.python.org/t/remove-outdated-tools-scripts-scripts/19571/22.

**Review by gvanrossum — APPROVED** [review]
I might preserve these too (in the same repo) but not right now. Go ahead with the removal though.

**Comment by vstinner** [comment]
> I might preserve these too (in the same repo) but not right now. Go ahead with the removal though.

Once most scripts will be removed and a copy of these scripts will be created, we can document the removal in What's New In Python 3.12 with a link to this copy.

**Review by gpshead — APPROVED** [review]

**Review by gvanrossum — APPROVED** [review]
Sure. I expect that we may be removing one or two scripts that are still depended on by some platform or workflow, but that's okay -- we'll just restore those when we get complaints.

**Comment by gvanrossum** [comment]
And yes, these are now also in my https://github.com/gvanrossum/old-demos repo.

File: Misc/NEWS.d/next/Tools-Demos/2022-09-30-14-30-12.gh-issue-97669.gvbgcg.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Remove outdated example scripts. Copy them from Python 3.11 if you still
+need them. Patch by Victor Stinner.
```
Comment (ezio-melotti) [inline Misc/NEWS.d/next/Tools-Demos/2022-09-30-14-30-12.gh-issue-97669.gvbgcg.rst]: Should this mention Guido's repo too?

**Comment by vstinner** [comment]
I completed to doc to mention the removal in What's New in Python 3.12 with a link to https://github.com/gvanrossum/old-demos

**Comment by vstinner** [comment]
Ok, I removed a first batch of outdated scripts.

**Pull request merged**